packages/api-explorer/src/index.jsx

```diff
@@ -133,7 +133,6 @@ class ApiExplorer extends React.Component {
         appearance={this.props.appearance}
         language={this.state.language}
         oauth={this.props.oauth}
-        suggestedEdits={false}
```

Comment:
It's not used anymore?

Reply:
it was removed because we only used it to false

Reply:
Yeah, it's never been used